docs/index.rst

```diff
@@ -31,6 +31,16 @@
 
     /rst/fastdds_qos_profiles_manager_cli/cli
```

Comment:
This and the following headers look crop when documentation generated in html (see attached image)
Should we try to shorten somehow the names?

![image](https://user-images.githubusercontent.com/120394830/233639247-16df66b8-1037-4a83-b19e-6c853552e31e.png)

Reply:
@EduPonz I would like your opinion here

Reply:
Discuss internally. Project name is already displayed in the upper section so it could be removed.